packages/idempotency/tests/unit/persistence/DynamoDbPersistenceLayer.test.ts

```diff
@@ -395,19 +396,32 @@ describe('Class: DynamoDBPersistenceLayer', () => {
         expiryTimestamp: 0,
       });
       client.on(PutItemCommand).rejects(
-        new DynamoDBServiceException({
-          $fault: 'client',
+        new ConditionalCheckFailedException({
+          //$fault: 'client',
```

Comment:
How come this is commented out?

Reply:
keep getting this error and can't seem to figure out why because the property `$fault` is actually a member of the ConditionalCheckFailedException [class](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-dynamodb/Class/ConditionalCheckFailedException/):
```

Argument of type '{ $fault: string; $metadata: { httpStatusCode: number; requestId: string; }; message: string; Item: { id: { S: string; }; status: { S: string; }; expiration: { N: string; }; }; name: string; }' is not assignable to parameter of type 'ExceptionOptionType<ConditionalCheckFailedException, DynamoDBServiceException>'.
      Object literal may only specify known properties, and '$fault' does not exist in type 'ExceptionOptionType<ConditionalCheckFailedException, DynamoDBServiceException>'.
```

Reply:
Let's remove the line